README.md

```diff
@@ -27,53 +22,196 @@ ember install ember-a11y-testing
 
 ## Usage
 
+Setting up `ember-a11y-testing` in your tests can be done in one of two ways:
+
+1. A one-time setup in your tests/test-helper.js file using `setupGlobalA11yHooks`
+1. In individual tests using the `a11yAudit` test helper.
+
 ### aXe Options
 
 When using the `a11yAudit` helper, you can pass in `axe-core` options.
 These options are documented in the [axe-core API docs](https://www.deque.com/axe/axe-for-web/documentation/api-documentation/#options-parameter).
 The rule definitions are documented on [dequeuniversity.com/rules](https://dequeuniversity.com/rules/axe/3.2).
 
-### Testing Usage
+Each of the following sections individually details how to set aXe options for your tests.
+
+### `setupGlobalA11yHooks` Usage
+
+The `setupGlobalA11yHooks` function is intended to be imported and invoked a single time for your entire test suite.
+
+```ts
+export function setupGlobalA11yHooks(
+  shouldAudit: InvocationStrategy,
+  audit: (...args: any[]) => PromiseLike<void> = a11yAudit
+);
+```
+
+The `setupGlobalA11yHooks` function takes two parameters:
+
+- `shouldAudit`: A [predicate function](https://stackoverflow.com/a/1344021/769) that returns a `boolean` indicating whether or not to perform the audit.
+- `audit`: The audit function, which performs the `axe-core` audit, defaulting to `a11yAudit`. This allows you to potentially wrap the `a11yAudit` test helper with custom logic.
+
+To use, import and invoke the global setup function, passing in your specific invocation strategy:
+
+```js
+import Application from 'my-app/app';
+import config from 'my-app/config/environment';
+import { setApplication } from '@ember/test-helpers';
+import { start } from 'ember-qunit';
+import { setupGlobalA11yHooks } from 'ember-a11y-testing/test-support';
+import { customInvocationStrategy } from './custom-invocation-strategy';
+
+setApplication(Application.create(config.APP));
+
+setupGlobalA11yHooks(customInvocationStrategy);
+
+start();
+```
+
+Examples of invocation strategies can be found in [this](https://github.com/ember-a11y/ember-a11y-testing/blob/50ef5f8fff4aa91d7a85b9feee5b1ce9bf380df9/tests/acceptance/setup-global-a11y-hooks-test.ts#L32) [repository's](https://github.com/ember-a11y/ember-a11y-testing/blob/50ef5f8fff4aa91d7a85b9feee5b1ce9bf380df9/tests/acceptance/setup-global-a11y-hooks-test.ts#L81) [tests](https://github.com/ember-a11y/ember-a11y-testing/blob/50ef5f8fff4aa91d7a85b9feee5b1ce9bf380df9/tests/acceptance/setup-global-a11y-hooks-test.ts#L135).
+
+#### Setting Options with `setupGlobalA11yHooks`
```

Comment:
FWIW, I think that the `setRunOptions` API has nothing to do with `setupGlobalA11yHooks` specfiically. Any invocation of `a11yAudit` will use these options, not just the ones done by `setupGlobalA11yHooks`...

Reply:
I think it relates more to the global hooks, but you're correct in that you _can_ use the APIs anywhere. You can still pass the `options` into the `a11yAudit` calls directly, which is what I was trying to disambiguate.